internal/cephfs/util.go

```diff
@@ -45,9 +44,7 @@ func execCommand(ctx context.Context, program string, args ...string) (stdout, s
 	cmd.Stdout = &stdoutBuf
 	cmd.Stderr = &stderrBuf
 
-	if klog.V(util.Debug).Enabled() {
-		klog.InfoDepth(1, util.Log(ctx, "cephfs: EXEC %s %s"), program, sanitizedArgs)
-	}
+	util.DebugLog(util.Debug, fmt.Sprintf(util.Log(ctx, "cephfs: EXEC %s %s"), program, sanitizedArgs))
```

Comment:
`util.DebugLog(util.Debug` is a little double. No need to pass the loglevel for a `DebugLog()` function, it should always be debug.

Reply:
Add other functions. like `util.InfoLog()`. That makes it really easy to use them. Try to make it `DebugLog("hello %s", name)` so there is no need to do the `fmt.Sprintf()` call when logging (it'll likely be used inside the `DebugLog()` function).